spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/OpenTelemetryConfigurations.java

```diff
@@ -105,10 +120,9 @@ Sampler otelSampler(TracingProperties properties) {
 		}
 
 		@Bean
-		@ConditionalOnMissingBean
 		SpanProcessor otelSpanProcessor(List<SpanExporter> spanExporter) {
 			return SpanProcessor.composite(spanExporter.stream()
-					.map((exporter) -> BatchSpanProcessor.builder(exporter).build()).collect(Collectors.toList()));
+					.map(exporter -> BatchSpanProcessor.builder(exporter).build()).collect(Collectors.toList()));
```

Comment:
`(exporter) ->` is Spring Boot's code style and this doesn't pass Checkstyle. Please build your changes before submitting them.